config/tests/samples/create/harness.go

```diff
@@ -504,7 +503,7 @@ func (h *Harness) CompareGoldenFile(p string, got string, normalizers ...func(s
 		if err := os.WriteFile(p, []byte(got), 0644); err != nil {
 			h.Fatalf("failed to write golden output %s: %v", p, err)
 		}
-		h.Errorf("wrote output to %s", p)
+		h.Logf("wrote output to %s", p)
```

Comment:
I think this should be Errorf because the golden files are different.  Can you revert this line & send separately if you do think it should be Logf?